portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,9 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  
 }
 
-#greeting-container {
+#Show-container {
```

Comment:
Nit: I don't know if I have ever seen CSS selectors start with a capital letter. I've always seen `lowercase-names-with-hyphens`. Whatever you want to go with, I would just be consistent :slightly_smiling_face: So if you want to stick with `Show-container` then I would update `#content` to `#Content`.